lisa/operating_system.py

```diff
@@ -346,7 +347,19 @@ def get_package_information(
     def get_repositories(self) -> List[RepositoryInfo]:
         raise NotImplementedError("get_repositories is not implemented")
 
-    def _install_packages(self, packages: List[str], signed: bool = True) -> None:
+    def remove_not_existed_packages(self, packages: List[str]) -> List[str]:
```

Comment:
Can we just ignore errors from the installation log? It would be much faster than check each package.

Reply:
I don't understand 'ignore errors from the installation log', is it 
try install, find failure, exclude the ones which cause the failure?

Reply:
No, it should only ignore errors if a package doesn't exist. Because the installation may include various package names for different distros together.

Reply:
no, like I code in xfstests,if install A,B,C,D in one command, and if C doesn't exist, then the entire process will be terminated, in my knowledge; but I will double check and see.

Reply:
ok, I see. Is there a way to check multiple packages in one command?

Reply:
I'll try to figure out.

Reply:
I didn't find a way to install them in one command.
Maybe we can just install them one by one without checking existed or not when ignore_non_exists set as true?

Reply:
We can check them like current way, if the ignore_non_exists is set to True. Install one by one is lower performance, and may miss other kind of errors too. In another hand, we shouldn't overuse this, we should try our best to match package names and distros.